src/loaders/lottie/tvgLottieInterpolator.cpp

```diff
@@ -127,7 +127,7 @@ float LottieInterpolator::progress(float t)
 
 void LottieInterpolator::set(const char* key, Point& inTangent, Point& outTangent)
 {
-    this->key = strdup(key);
+    if (key) this->key = strdup(key);
```

Comment:
yes, I mentioned in desc that this pr contains 2951. can rebase after 2951 is merged or you can rev only this one - this one is easier to test, since testing easing on sh=square is not so intuitive